Cargo.toml

```diff
@@ -12,11 +12,11 @@ repository = "https://github.com/pganalyze/pg_query.rs"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-itertools = "*"
-prost = "*"
-serde = { version = "*", features = ["derive"] }
-serde_json = "*"
-thiserror = "*"
+itertools = "0.10.3"
+prost = "0.10.4"
+serde = { version = "1.0.139", features = ["derive"] }
+serde_json = "1.0.82"
+thiserror = "1.0.31"
```

Comment:
crates.io doesn't allow dependencies to have a wildcard version requirement https://github.com/rust-lang/rfcs/pull/1241